core/web/jsonld/ArticleTrait.php

```diff
@@ -0,0 +1,209 @@
+<?php
+
+namespace luya\web\jsonld;
+
+/**
+ * JsonLd - Article Trait
+ *
+ * @see http://schema.org/Article
+ *
+ * @author Alex Schmid <alex.schmid@stud.unibas.ch>
+ * @since 1.0.1
+ */
+trait ArticleTrait
+{
+    use CreativeWorkTrait;
+
+    /**
+     * The actual body of the article.
+     *
+     * @var string
+     */
+    private $_articleBody;
+
+    /**
+     * @return string
+     */
+    public function getArticleBody()
+    {
+        return $this->_articleBody;
+    }
+
+    /**
+     * @param string $articleBody
+     * @return Article|ArticleTrait
+     */
+    public function setArticleBody($articleBody)
+    {
+        $this->_articleBody = $articleBody;
+        return $this;
+    }
+
+    /**
+     * Articles may belong to one or more 'sections' in a magazine or newspaper, such as Sports, Lifestyle, etc.
```

Comment:
@JohnnyMcWeed The private will not be shown anywhere (not in IDE nor on luya.io/api) so the information should be stored on the getter method. You can even fully remove any phpdoc for the private var. (for example: https://github.com/luyadev/luya/blob/master/core/tag/BaseTag.php)

Reply:
You're right. I'll change that for all classes in the next few days.